petpals/forum/post/templates/edit_post.html

```diff
@@ -0,0 +1,66 @@
+{% extends '_layout.html' %}
+{% block title %}New Post{% endblock %}
+{% set active_page = 'post_form' %}
+
+{% block body %}
+
+
+<section class="bg-image background-paws">
+  <div class="container py-5">
+    <div class="card rounded-3 border border-info border-6">
+      <div class="card-body p-md-5 mx-md-4">
+        <div class="text-center">
+          <img src="/static/images/logo.jpeg" alt="Pet Pals" class="mb-1" />
+          <h5 class="mb-4">Where your Pet meets new Pals</h5>
+        </div>
+        <h1 class="text-center mb-5">Create New Post Form</h1>
+
+        {% if current_user.is_authenticated %}
```

Comment:
if and endif not needed as the user _has_ to be authenticated to access this page.